plugins/modules/monitoring/datadog/datadog_monitor.py

```diff
@@ -216,7 +221,8 @@ def main():
             require_full_window=dict(required=False, default=None, type='bool'),
             new_host_delay=dict(required=False, default=None),
             evaluation_delay=dict(required=False, default=None),
-            id=dict(required=False)
+            id=dict(required=False),
+            include_tags=dict(required=False, default=True, type='bool')
```

Comment:
```suggestion
            include_tags=dict(required=False, default=True, type='bool'),
```
Then the next diff will be cleaner (and not has to touch this line).